<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connexion</class>
 <widget class="QWidget" name="connexion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelH1">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>146</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Connectez-vous&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassword">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>290</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassword">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>76</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mot de passe</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFeature">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>64</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Vous voulez :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxFeature">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>267</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Ajouter à la base de données</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Supprimer de la base de données après une vente</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Consulter la base de données</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButtonConnect">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Se connecter</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
